[DOCU-3172] MinK: Overview + UI screenshot (#5432)
Kong/docs.konghq.com: app/konnect/mesh-manager/service-mesh.md

---
title: Create a service mesh
content_type: how-to
---

Creating a fully-functioning {{site.mesh_product_name}} global control plane in {{site.konnect_short_name}} involves the following steps:

1. Create the global control plane in {{site.konnect_short_name}}.
1. Add and configure a zone for your control plane.
1. Configure `kumactl` to connect to your global control plane.
1. Add services to your service mesh.

### Prerequisites

* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)

> * [Download the latest version of {{site.mesh_product_name}}](/mesh/latest/production/install-kumactl/)

Author: ```suggestion
* [The latest version of {{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
```

---
title: Create a service mesh
content_type: how-to
---

> Creating a fully-functioning {{site.mesh_product_name}} global control plane in {{site.konnect_short_name}} involves the following steps:

Reviewer: ```suggestion
Creating a fully-functioning {{site.mesh_product_name}} deployment in {{site.konnect_short_name}} involves the following steps:
```

1. Create the global control plane in {{site.konnect_short_name}}.
1. Add and configure a zone for your control plane.
1. Configure `kumactl` to connect to your global control plane.
1. Add services to your service mesh.

### Prerequisites

* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)

### Create and configure the global control plane

> 1. From the left navigation menu in {{site.konnect_short_name}}, open [MESH MANAGER ICON HERE] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).

Reviewer 2: ```suggestion
1. From the left navigation menu in {{site.konnect_short_name}}, open [/assets/images/icons/konnect/icn-mesh-manager.svg] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).
```

---
title: Create a service mesh
content_type: how-to
---

Creating a fully-functioning {{site.mesh_product_name}} global control plane in {{site.konnect_short_name}} involves the following steps:

1. Create the global control plane in {{site.konnect_short_name}}.
1. Add and configure a zone for your control plane.
1. Configure `kumactl` to connect to your global control plane.
1. Add services to your service mesh.

### Prerequisites

* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)

> * Mutual TLS only: If you plan to automatically encrypt [mTLS](/mesh/latest/policies/mutual-tls/) traffic for all the services in your service mesh, you will need the mTLS certificate name and authority.
> * Logging only: If you plan to add a logging backend to log traffic via the TrafficLog policy, you will need the IP address and format (WHAT IS THIS?) of your logging backend. [WHICH BACKENDS ARE AVAILABLE?]
> * Tracing only: If you plan to add a tracing backend to ___ using the TrafficTrace policy, you will need the sampling [WHAT IS THIS?] and the url of your backend. [WHICH BACKENDS ARE AVAILABLE?]
> * Metrics only: If you want to port metrics from your data plane, you will need the data plane port and path of your metrics backend. [WHICH BACKENDS ARE AVAILABLE?]
> >>>>>>> f4996ec469bff82b70f8e9b2a93f0b15c12f6c7b

Author: ```suggestion
```

---
title: Create a service mesh
content_type: how-to
---

Creating a fully-functioning {{site.mesh_product_name}} global control plane in {{site.konnect_short_name}} involves the following steps:

1. Create the global control plane in {{site.konnect_short_name}}.
1. Add and configure a zone for your control plane.
1. Configure `kumactl` to connect to your global control plane.
1. Add services to your service mesh.

### Prerequisites

* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)

> * [Download the latest version of {{site.mesh_product_name}}](/mesh/latest/production/install-kumactl/)

Author: ```suggestion
```

---
title: Create a service mesh
content_type: how-to
---

Creating a fully-functioning {{site.mesh_product_name}} global control plane in {{site.konnect_short_name}} involves the following steps:

1. Create the global control plane in {{site.konnect_short_name}}.
1. Add and configure a zone for your control plane.
1. Configure `kumactl` to connect to your global control plane.
1. Add services to your service mesh.

### Prerequisites

* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)

> <<<<<<< HEAD
> * The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
> =======

Author: ```suggestion
* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
```
